src/App.js

```diff
@@ -1,13 +1,22 @@
 import React from "react";
-
-import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Bookings from "./Bookings";
+import Footer from "./Footer";
 
 const App = () => {
+  const FooterInfo = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer Info={FooterInfo} />
```

Comment:
This looks good, but the React *convention* is to start prop names with a lower-case letter. Only component names should start with an upper-case letter.